bin/git-fork

```diff
@@ -34,10 +34,15 @@ fi
 curl -X POST -u "$user" "https://api.github.com/repos/$owner/$project/forks"
 [ $? = 0 ] || abort "fork failed"
 
-# clone forked repo into current dir
-git clone "https://github.com/$user/$project" "$project"
+if [ "$origin" = true ]; then
+    git remote rename origin upstream
+    git remote add origin "https://github.com/$user/$project"
```

Comment:
We need to do a `git fetch --all` here to update the references and make the new `origin` "visible" so to speak.